maxitg/coherent-enhancement
# Supersymmetry simulations

## Changes

* Supersymmetry model equations were incomplete and incorrect. Changed to m = 1, so that there are two, rather than four superfields, which is the model used in simulations.
* Added 4 figures for supersymmetry: potential, tensor-to-scalar decay ratio vs. decay constant, enhancement, and comparison of static and dynamic effective decay constants. The figures and their captions are now generated with a `.wls` file. Thus, deleted explicit `.pdf` figures.
* Added definition of dynamic decay constant $f_{eH}$ in section 3.
* Added explicit Planck masses in sections 3 and 4.
* Added an explanation of why $\mu$ and $\lambda$ disappear from the slow potential.

## Commands & tests

To build,
* First, `cd` into the repository directory.
* Run `./provision.sh`. That will install necessary LaTeX packages.
* Run `./build.sh`. That will produce figures, their captions, and then `pdflatex` the paper PDF. Note, this currently takes about 30 minutes, because it runs supersymmetry simulations from scratch. It will take even longer in the future after other sections are added. Note, Wolfram Mathematica 11.3 is required, and Wolfram Script must be installed.
* Check figures and their captions are generated correctly.
